src/es/components/molecules/facts/Facts.js

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+export default class Facts extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+  }
+
+  disconnectedCallback () {}
+
+  shouldRenderCSS () {
+    return !this.root.querySelector(
+      `:host > style[_css], ${this.tagName} > style[_css]`
+    )
+  }
+
+  renderCSS () {
+    this.css = /* css */ `
+        :host {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          padding: 4rem 0 5rem;
+        }
+
+        :host h2 {
+          font-size: 2.25rem;
+          line-height: 2.5rem;
+          font-weight: 500;
+          margin: 0;
+        }
+
+        :host .facts {
+          display: grid;
+          grid-template-columns: 33.33% 33.33% 33.33%;
+          column-gap: 1.5rem;
+        }
+
+        :host h2 + .facts {
+          margin-top: 2rem;
+        }
+
+        :host .facts + ks-a-button {
+          margin-top: 3rem;
+        }
+
+        :host .fact {
+          background-color: var(--mdx-base-color-grey-100);
```

Comment:
doesn't figma supply a variable specific for this background? We hope that it can be automatically changed between different corp styles. @tdesero , do you know, if they have a specific variable?

Reply:
jepp this one should work: var(--mdx-sys-color-accent-6-subtle2).
But I think this will unfortunately not be consistent throughout brands: For IBAW another shade for this component was requested.

Reply:
ok, then we add a specific variable in the variables custom file later once we start with IBAW